src/parser/jobs/blu/modules/Swiftcast.tsx

```diff
@@ -0,0 +1,8 @@
+import {Trans} from '@lingui/react'
+import {DataLink} from 'components/ui/DbLink'
+import {Swiftcast as CoreSwiftcast} from 'parser/core/modules/Swiftcast'
+import React from 'react'
+
+export class Swiftcast extends CoreSwiftcast {
+	override suggestionContent = <Trans id="blu.swiftcast.suggestion.content">Cast a spell with <DataLink action="SWIFTCAST" /> before it expires.</Trans>
```

Comment:
Do we need this suggestion override? The core one gives the additional context about using it for movement or weaving that this one lacks.

Reply:
With the exception of the RDM module, this is actually something cargo-culted by every other job!

The underlaying issue is that simply using the core module directly makes `tsc` unhappy[0], so you need to override the class; everyone likely just copypasted the same class override that also happens to override the core suggestion.

Should be patched as of 45b06261776e1e69de63ac4eda032075042b6abe 

[0]
```
; tsc
src/parser/jobs/blu/index.tsx:16:17 - error TS2322: Type 'Promise<typeof import("./src/parser/jobs/blu/modules/index")>' is not assignable to type 'Promise<{ default: (typeof Injectable)[]; }>'.
  Type 'typeof import("./src/parser/jobs/blu/modules/index")' is not assignable to type '{ default: (typeof Injectable)[]; }'.
    Types of property 'default' are incompatible.
      Type '(typeof Tincture | typeof Swiftcast | typeof AlwaysBeCasting | typeof BLURaidBuffs | typeof ColdFog | typeof Defensives | typeof DoTs | typeof DroppedBuffs | typeof GeneralCDDowntime | typeof Interrupts | typeof BLUOverheal | typeof RevengeBlast | typeof TripleTrident | typeof BLUWeaving)[]' is not assignable to type '(typeof Injectable)[]'.
        Type 'typeof Tincture | typeof Swiftcast | typeof AlwaysBeCasting | typeof BLURaidBuffs | typeof ColdFog | typeof Defensives | typeof DoTs | typeof DroppedBuffs | typeof GeneralCDDowntime | typeof Interrupts | typeof BLUOverheal | typeof RevengeBlast | typeof TripleTrident | typeof BLUWeaving' is not assignable to type 'typeof Injectable'.
          Type 'typeof Swiftcast' is not assignable to type 'typeof Injectable'.
            Cannot assign an abstract constructor type to a non-abstract constructor type.

16  modules: () => import('./modules' /* webpackChunkName: "jobs-blu" */),
                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

  src/parser/core/Meta.tsx:9:22
    9 type ModulesLoader = () => Promise<{default: Array<typeof Injectable>}>
                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    The expected type comes from the return type of this signature.


Found 1 error.

```